src/templates/utils-common.js

```diff
@@ -120,6 +121,9 @@ export function getDomainFromLocale (localeCode, req, { normalizedLocales }) {
 // Lookup the `differentDomain` origin associated with given locale.
   const lang = normalizedLocales.find(locale => locale.code === localeCode)
   if (lang && lang.domain) {
+    if (hasProtocol(lang.domain, true)) {
```

Comment:
I think that it would be better to change `toAbsoluteUrl` function to accept `//` URLs because it could be used both on development and production deployments.

However, `//` style URLs are currently marked as anti-pattern because it gives browser opportunity to select on its own instead of forcing SSL (which is better).

Basically, I think that disabling accepting  `// ` style URLs is a way to go.

Thanks!

Reply:
I agree